konan/konan.properties

```diff
@@ -26,7 +26,7 @@ airplaneMode = false
 # if true, ignores Xcode version check.
 ignoreXcodeVersionCheck = false
 
-minimalXcodeVersion=11.0
+minimalXcodeVersion=12.0
```

Comment:
Is it? Does compiler work with Xcode 11?

Reply:
It will, but platform libraries will be inconsistent with actually available SDKs.

Reply:
Will this be a problem if the code doesn't use new APIs?

Reply:
No, but it will be a problem from UX point of view (declaration is available in IDE when it actually isn't). OTOH, I agree with you that forcing users to switch to Xcode 12 isn't a good move.